dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/DotRestHighLevelClientProvider.java

```diff
@@ -69,6 +69,8 @@
 
     private RestHighLevelClient client;
 
+    private NativeHttpRestClient nativeClient;
```

Comment:
I think this should be a new Client Provider (i.e. `NativeHttpRestClientProvider`) instead of modifying this one.